includes/course-theme/class-sensei-course-theme-templates.php

```diff
@@ -121,7 +250,8 @@ private function load_file_templates() {
 			return;
 		}
 
-		$base_path = Sensei_Course_Theme::instance()->get_course_theme_root() . '/templates/';
+		$template = $this->get_active_block_template();
+		$title    = isset( $template['title'] ) ? $template['title'] : $template['name'];
```

Comment:
We can use the null coalescing operator now:
```suggestion
		$title    = $template['title'] ?? $template['name'];
```

Reply:
Update to use `??` operator!